websites/T/TETR.IO/dist/metadata.json

```diff
@@ -4,18 +4,25 @@
     "name": "Rare Akuma ☆",
     "id": "250044591686811669"
   },
+  "contributors": [
+    {
+      "name": "Hyruled",
+      "id": "600754443318853632"
+    }
+  ],
```

Comment:
Didn't see him doing pull request about doing things so delete it.

Reply:
oh, I see, np